A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux, `bionic` branch
```
CMake Error at test/qmltest/modules/TestUtil/CMakeLists.txt:18 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Notifications/CMakeLists.txt:24 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Notifications/CMakeLists.txt:33 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Notifications/Mocks/CMakeLists.txt:7 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Launcher/CMakeLists.txt:32 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Application/CMakeLists.txt:23 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Scopes/CMakeLists.txt:42 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

```
CMake Error at test/qmltest/mocks/plugins/Unity/Scopes/CMakeLists.txt:50 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete

--- a/test/qmltest/mocks/plugins/Unity/Application/CMakeLists.txt
+++ b/test/qmltest/mocks/plugins/Unity/Application/CMakeLists.txt
@@ -20,7 +20,7 @@ set(ApplicationMocks_SOURCES
 
 add_library(ApplicationMocks SHARED ${ApplicationMocks_SOURCES})
 
-qt5_use_modules(ApplicationMocks Core Gui)
+target_link_libraries(ApplicationMocks Qt5::Core Qt5::Gui)
 
 set(TestApplicationPlugin_SOURCES
     TestApplicationPlugin.cpp
@@ -28,7 +28,7 @@ set(TestApplicationPlugin_SOURCES
 
 add_library(TestApplicationPlugin MODULE ${TestApplicationPlugin_SOURCES})
 
-qt5_use_modules(TestApplicationPlugin Core Gui Quick)
+target_link_libraries(TestApplicationPlugin Qt5::Core Qt5::Gui Qt5::Quick)
 
 target_link_libraries(TestApplicationPlugin ApplicationMocks)
 

--- a/test/qmltest/mocks/plugins/Unity/Launcher/CMakeLists.txt
+++ b/test/qmltest/mocks/plugins/Unity/Launcher/CMakeLists.txt
@@ -29,7 +29,8 @@ set(LauncherMocks_SOURCES
 
 add_library(LauncherMocks SHARED ${LauncherMocks_SOURCES})
 
-qt5_use_modules(LauncherMocks Core Gui)
+#find_package(Qt5Gui REQUIRED)
+target_link_libraries(LauncherMocks Qt5::Core Qt5::Gui)
 
 set(TestLauncherPlugin_SOURCES
     TestLauncherPlugin.cpp
@@ -37,7 +38,7 @@ set(TestLauncherPlugin_SOURCES
 
 add_library(TestLauncherPlugin MODULE ${TestLauncherPlugin_SOURCES})
 
-qt5_use_modules(TestLauncherPlugin Core Quick)
+target_link_libraries(TestLauncherPlugin Qt5::Core Qt5::Quick)
 
 target_link_libraries(TestLauncherPlugin LauncherMocks)
 

--- a/test/qmltest/mocks/plugins/Unity/Notifications/CMakeLists.txt
+++ b/test/qmltest/mocks/plugins/Unity/Notifications/CMakeLists.txt
@@ -21,7 +21,7 @@ set(NotificationsMocks_SOURCES
 
 add_library(NotificationsMocks SHARED ${NotificationsMocks_SOURCES})
 
-qt5_use_modules(NotificationsMocks Core)
+target_link_libraries(NotificationsMocks Qt5::Core)
 
 set(TestNotificationsPlugin_SOURCES
     TestNotificationsPlugin.cpp
@@ -29,7 +29,7 @@ set(TestNotificationsPlugin_SOURCES
 
 add_library(TestNotificationsPlugin MODULE ${TestNotificationsPlugin_SOURCES})
 
-qt5_use_modules(TestNotificationsPlugin Core Quick)
+target_link_libraries(TestNotificationsPlugin Qt5::Core Qt5::Quick)
 
 target_link_libraries(TestNotificationsPlugin NotificationsMocks)
 

--- a/test/qmltest/mocks/plugins/Unity/Notifications/Mocks/CMakeLists.txt
+++ b/test/qmltest/mocks/plugins/Unity/Notifications/Mocks/CMakeLists.txt
@@ -4,7 +4,9 @@ set(MockNotificationsPlugin_SOURCES
 
 add_library(MockNotificationsPlugin MODULE ${MockNotificationsPlugin_SOURCES})
 
-qt5_use_modules(MockNotificationsPlugin Core Quick)
+#find_package(Qt5Core REQUIRED)
+#find_package(Qt5Quick REQUIRED)
+target_link_libraries(MockNotificationsPlugin Qt5::Core Qt5::Quick)
 
 target_link_libraries(MockNotificationsPlugin NotificationsMocks)
 

--- a/test/qmltest/mocks/plugins/Unity/Scopes/CMakeLists.txt
+++ b/test/qmltest/mocks/plugins/Unity/Scopes/CMakeLists.txt
@@ -39,7 +39,7 @@ set(ScopesMocks_SOURCES
 
 add_library(ScopesMocks SHARED ${ScopesMocks_SOURCES})
 
-qt5_use_modules(ScopesMocks Core)
+target_link_libraries(ScopesMocks Qt5::Core)
 
 set(TestScopesPlugin_SOURCES
     TestScopesPlugin.cpp
@@ -47,7 +47,7 @@ set(TestScopesPlugin_SOURCES
 
 add_library(TestScopesPlugin MODULE ${TestScopesPlugin_SOURCES})
 
-qt5_use_modules(TestScopesPlugin Core Quick)
+target_link_libraries(TestScopesPlugin Qt5::Core Qt5::Quick)
 
 target_link_libraries(TestScopesPlugin ScopesMocks)
 

--- a/test/qmltest/modules/TestUtil/CMakeLists.txt
+++ b/test/qmltest/modules/TestUtil/CMakeLists.txt
@@ -15,7 +15,7 @@ include_directories(
 
 add_library(TestUtilQml MODULE ${TestUtilQML_SOURCES})
 
-qt5_use_modules(TestUtilQml Core Quick)
+target_link_libraries(TestUtilQml Qt5::Core Qt5::Quick)
 
 # copy qmldir file into build directory for shadow builds
 file(GLOB QML_JS_FILES qmldir *.js *.qml)

